.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2
+jobs:
+  build:
+    docker:
+        - image: kaikuehne/pyenv:alpine3.10
+          environment:
+            PATH: "/root/pyenv/bin:/root/pyenv/shims:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin"
+            PYENV_ROOT: "/root/pyenv"
+    working_directory: /tmp/app
+    steps:
+        - checkout
+        - setup_remote_docker
+        - restore_cache:
+            keys:
+                - v1-project-{{ arch }}-{{ checksum "tox.ini" }}
+                - v1-project-
+        - run:
+            command: |
+                pyenv --version
+                pyenv install --list
+                pyenv install --keep --skip-existing 2.7.16
+                pyenv install --keep --skip-existing 3.7.4
+                pyenv rehash
+                pyenv versions
+                pyenv local  2.7.16 3.7.4
+                pip install --upgrade pip
+                pip install tox tox-pyenv
+            shell: /bin/bash
+        - save_cache:
+              key: v1-project-{{ arch }}-{{ checksum "tox.ini" }}
+              paths:
+                  - /root/pyenv
+        - run:
+            command: tox
```

Comment:
This workflow needs a lot of work, but it works for now.  It's copied from my bad attempt at getting percy working with circleci.